modules/filters.js

```diff
@@ -80,6 +80,10 @@ function timeago(date) {
     return moment(date).fromNow();
 }
 
+function addTime(date, amount, unit) {
```

Comment:
Will this make me live longer?

`addTime(now()), '25', 'years')`

Reply:
Sadly all you've done there is made your age `3025` 👴